mockdata/generate_customers.js

```diff
@@ -0,0 +1,28 @@
+const faker = require('faker');
+const fs = require('fs')
+
+function generateCustomers() {
+
+  let customers = []
+
+  for (let id = 1; id <= 50; id++) {
+    let customerid = faker.random.number({ min: 1, max: 50 });
+    let productid = faker.random.number({ min: 1, max: 100 });
```

Comment:
I'm a little confused why you have product ids and purchased dates in the customers document. If a customer can have many reviews, shouldn't these be arrays of ids?